socorro/external/postgresql/connection_context.py

```diff
@@ -6,10 +6,14 @@
 import contextlib
 import psycopg2
 import psycopg2.extensions
+import urlparse
```

Comment:
Ultrapicky, a good convention is to separate python standard lib from third-party libs. Can you move this to appear just after `import contextlib` and then add an extra space. The `import psycopg2` stuff belongs next to the `from configman.config_manager...` lines.